package.json

```diff
@@ -4,25 +4,30 @@
   "displayName": "SwiftLint",
   "description": "Use SwiftLint to lint Swift files",
   "publisher": "shinnn",
-  "repository": "https://github.com/shinnn/vscode-standard",
-  "homepage": "https://github.com/shinnn/vscode-standard#readme",
-  "bugs": "https://github.com/shinnn/vscode-standard/issues",
+  "repository": "https://github.com/shinnn/vscode-swiftlint",
+  "homepage": "https://github.com/shinnn/vscode-swiftlint#readme",
+  "bugs": "https://github.com/shinnn/vscode-swiftlint/issues",
   "license": "MIT",
   "icon": "icon.svg",
   "galleryBanner": {
     "color": "#F2F2F2",
     "theme": "light"
   },
   "engines": {
-    "vscode": "^0.10.1"
+    "vscode": "^1.17.0"
   },
   "categories": [
     "Linters"
   ],
   "activationEvents": [
     "onLanguage:swift"
   ],
-  "main": "extension.js",
+  "keywords": [
+    "swift",
+    "swiftlint",
+    "linter"
+  ],
+  "main": "src/extension.js",
```

Comment:
> ```diff
> - "main": "extension.js"
> + "main": "src/extension.js"
> ```

Can I ask why?

Reply:
Sure, it was hard to differentiate source code for extension vs all the other files in the root folder. I don't have a strong preference personally, can move it back if you'd prefer.